schema.json

```diff
@@ -22,15 +24,25 @@
         },
         "version": {
           "type": "string",
-          "description": "The version of the collection to pull",
+          "description": "The version of the collection to pull.",
           "pattern": "^\\d+[.]\\d+([.]\\d+)?$"
         },
         "style": {
           "type": "string",
-          "description": "Style to apply to the collection"
+          "description": "Style to apply to the collection."
+        },
+        "uuid": {
+          "type": "string",
+          "description": "UUID to serve as another point fo reference.",
+          "format": "uuid"
         }
       },
-      "required": ["server", "collection_id", "version", "style"],
+      "required": [
+        "server",
+        "collection_id",
+        "version",
+        "style"
```

Comment:
Do we want to add uuid as a required field? 🤷

Reply:
I'm a bit confused, wouldn't `additionalProperties` fail to validate the file above since it contains uuids and they aren't in the required list?

Reply:
Curious, why would uuid be considered a required field? @tomjw64 

From my understanding (and conclusion I came up with myself haha) is that the current required fields are required because it's a must for neb.

Reply:
I suppose it is not 100% needed for neb to do its work. I think it might be good for completeness, though, since it is a server-independent way of identifying a book.